src/evaluate/evaluator/utils.py

```diff
@@ -16,3 +16,13 @@ def __getitem__(self, i):
 
     def __iter__(self):
         return (self.dataset[i][self.key] for i in range(len(self)))
+
+
+def choose_split(data):
```

Comment:
One concern I have here is about naming of the splits - it seems customary to use `test` / `validation` / `train` in huggingface code but I've seen other variations especially for `validation` - e.g. `val` or `dev`. I wonder if it would make sense to test for those as well?

Reply:
Good point. I've changed it to look for splits in this order: `["test", "validation", "val", "valid", "train", "dev"]`, and raise an error if none of these is found. If the dataset they're using doesn't follow any of these canonical namings and they insist on passing it in by name (as opposed to an instantiated Dataset object) they should be forced to explicitly pick a split instead of having `evaluate` guess which one is the best according to size or another heuristic.

Reply:
In my experience `"dev"` is another name for validation so I'd put it before `"train"`

Reply:
Moved, thank you!